_inc/client/components/jetpack-termination-dialog/dialog.jsx

```diff
@@ -125,13 +127,14 @@ class JetpackTerminationDialog extends Component {
 	};
 
 	renderFeatures() {
-		const { isDevSite: siteIsDev, purpose, siteBenefits } = this.props;
+		const { isDevSite: siteIsDev, purpose, siteBenefits, connectedPlugins } = this.props;
 
-		return siteBenefits ? (
+		return siteBenefits && connectedPlugins ? (
```

Comment:
Does this mean we won't display the modal at all when there are no connected plugins?

Reply:
Both `siteBenefits` and `connectedPlugins` are `null` by default, so the condition makes the dialog wait until both variables receive their values.
Empty array casts to `true`, so even if both `siteBenefits` and `connectedPlugins` are empty, the "Disconnect" modal will still display after AJAX requests are completed.